JS.md

```diff
@@ -166,6 +166,29 @@ Otherwise, this function performs synchronous compilation of the `BufferSource`:
   the validated `Ast.module`.
 * On failure, a new `WebAssembly.CompileError` is thrown.
 
+### `WebAssembly.Module.exports`
```

Comment:
Yes, that's the pattern since ES5. The difference is that it enables a more centralised way for policing reflection. Not that I'd have an immediate use case for that with modules, but we can just as well follow the pattern.